packages/snaps-utils/src/manifest/validation.ts

```diff
@@ -21,6 +21,13 @@ import { CronjobSpecificationArrayStruct } from '../cronjob';
 import { NamespacesStruct } from '../namespace';
 import { NameStruct, NpmSnapFileNames, VersionStruct } from '../types';
 
+// BIP-43 purposes that cannot be used for entropy derivation. These are in the
+// string form, ending with `'`.
+const FORBIDDEN_PURPOSES: string[] = [
+  // SIP-6 magic value.
+  `1399742832'`,
```

Comment:
Could this be a shared constant so we don't have to have it defined multiple places?

Reply:
Yes.